### PR TITLE
gh-1178 show key details after generating it

### PR DESCRIPTION
**Pull request by sche**

### CODE REVIEW

File: Multisig/Logic/Models/PrivateKey/PrivateKey.swift [hunk 1]
```diff
@@ -26,7 +26,7 @@ struct PrivateKey {
     }
 
     /// needed for PrivateKeyController.migrateLegacySigningKey()
-    var keyData: Data {
+    var legacyKeyData: Data {
```
Comment (DmitryBespalov) [inline Multisig/Logic/Models/PrivateKey/PrivateKey.swift]: well, you know, sorry now to revert my previous comment - we need this data actually to show the private key in the export screen, I would not change it now.

File: Multisig/UI/Settings/OwnerKeyManagement/GenerateOwnerKey/OnboardingGenerateKeyViewController.swift [hunk 2]
```diff
@@ -61,12 +61,32 @@ class OnboardingGenerateKeyViewController: UITableViewController {
...
+                  let keyInfo = try? KeyInfo.keys(addresses: [privateKey.address]).first,
+                  let sceneDelegate = UIApplication.shared.connectedScenes.first?.delegate as? SceneDelegate else {
+                return
+            }
+
             if App.shared.auth.isPasscodeSet {
-                App.shared.snackbar.show(message: "Owner key successfully imported")
-                vc.dismiss(animated: true, completion: nil)
+                vc.dismiss(animated: false) {
+                    App.shared.snackbar.show(message: "Owner key successfully imported")
```
Comment (DmitryBespalov) [inline Multisig/UI/Settings/OwnerKeyManagement/GenerateOwnerKey/OnboardingGenerateKeyViewController.swift]: the snackbar message must be different:

`The key successfully created. Add it to the safe using the desktop app and then restart mobile app.`

File: Multisig/UI/Settings/OwnerKeyManagement/GenerateOwnerKey/OnboardingGenerateKeyViewController.swift [hunk 3]
```diff
@@ -61,12 +61,32 @@ class OnboardingGenerateKeyViewController: UITableViewController {
...
+                    sceneDelegate.show(OwnerKeyDetailsViewController(keyInfo: keyInfo))
+                }
             } else {
-                let createPasscodeViewController = CreatePasscodeViewController()
+                let createPasscodeViewController = CreatePasscodeViewController {
+                    App.shared.snackbar.show(message: "Owner key successfully imported")
```
Comment (DmitryBespalov) [inline Multisig/UI/Settings/OwnerKeyManagement/GenerateOwnerKey/OnboardingGenerateKeyViewController.swift]: Snackbar message: `The key successfully created. Add it to the safe using the desktop app and then restart mobile app.`

**Review by DmitryBespalov — COMMENTED** [review]
some more left to do

File: Multisig/Logic/Models/PrivateKey/PrivateKey.swift (hunk 1 above)
Comment (sche) [reply Multisig/Logic/Models/PrivateKey/PrivateKey.swift]: ok

File: Multisig/UI/Settings/OwnerKeyManagement/GenerateOwnerKey/OnboardingGenerateKeyViewController.swift (hunk 2 above)
Comment (sche) [reply Multisig/UI/Settings/OwnerKeyManagement/GenerateOwnerKey/OnboardingGenerateKeyViewController.swift]: ok

File: Multisig/UI/Settings/OwnerKeyManagement/GenerateOwnerKey/OnboardingGenerateKeyViewController.swift (hunk 3 above)
Comment (sche) [reply Multisig/UI/Settings/OwnerKeyManagement/GenerateOwnerKey/OnboardingGenerateKeyViewController.swift]: ok

**Review by DmitryBespalov — APPROVED** [review]

**Pull request merged**